config/locales/en/pvb2-mailers.yml

```diff
@@ -106,6 +106,9 @@ en:
         Your visit can't take place as the prisoner has a restriction.
       child_protection_issues:
         Your visit can't take place as the prisoner has a restriction.
+      duplicate_visit:
+        This is a duplicate visit request, you'll have received an email
+        notifying you if the original request was approved or rejected.
```

Comment:
You have already requested a visit with these details. You will have received an email regarding your original visit request, containing link to check the status of your request.